README.md

```diff
@@ -145,17 +142,17 @@ for try await conversation in client.conversations.stream() {
 }
 ```
 
-#### Start a new conversation
+### Start a new conversation
 
 You can create a new conversation with any Ethereum address on the XMTP network.
 
 ```swift
 let newConversation = try await client.conversations.newConversation(with: "0x3F11b27F323b62B159D2642964fa27C46C841897")
 ```
 
-#### Sending messages
+### Send messages
 
-To be able to send a message, the recipient must have already started their Client at least once and consequently advertised their key bundle on the network. Messages are addressed using wallet addresses. The message payload must be a plain string.
+To be able to send a message, the recipient must have already started a client at least once and consequently advertised their key bundle on the network. Messages are addressed using account addresses. The message payload must be a plain string.
```

Comment:
I wonder if `created` would be a better term to use here than `started`, since the method in the code is `Client.create` and the word `start` doesn't really appear anywhere in the SDK.

```suggestion
To be able to send a message, the recipient must have already created a client at least once and consequently advertised their key bundle on the network. Messages are addressed using account addresses. The message payload must be a plain string.
```

Reply:
Yes, please - this is a great callout, @nakajima! Let's do it - and I'll note it as an edit to make for other README work.